adhocracy4/comments/services.py

```diff
@@ -0,0 +1,8 @@
+from .models import Comment
+
+
+def delete_comments(contenttype, pk):
+    comments = Comment.objects.all().filter(
+        content_type=contenttype, object_pk=pk)
+    for comment in comments:
+        comment.delete()
```

Comment:
Ok. But where is it called? More specifically: In which cases is this used and in which cases is the code in adhocracy4/comments/signals.py used?